webapp/src/Controller/Jury/ExecutableController.php

```diff
@@ -86,65 +88,13 @@ public function __construct(
     /**
      * @Route("", name="jury_executables")
      * @param Request $request
```

Comment:
We can also get rid of this

Reply:
Why is the param not needed?

Reply:
It’s already in the type of the function arg. Only when that’s not possible we should add it

Reply:
https://softwareengineering.stackexchange.com/questions/365176/are-docblock-typehints-redundant-when-using-strict-typing

Ok... I understand, will remove them